develop/parachain-devs/parachain-launch/chain-spec.md

```diff
@@ -0,0 +1,113 @@
+
+1. Utilizing `polkadot-parachain` or compiled node binary
+2. Utilizing `chain-spec-builder`
+
+If you are creating a one-off network for development, testing, or demonstration purposes, you might want a fully customized chain specification.
+To create a completely customized chain spec, you can export the default chain spec to JSON format, then edit the fields in the JSON file.
+For example, you can use the `build-spec` sub-command to export the chain specification to a JSON file:
+
+### Using Node Binary
+
+
+!!!warning "Be aware of the Omninode"
+    As mentioned, there are two methods for generating a chain specification. The preferred method is [`chain-spec-builder`](https://paritytech.github.io/polkadot-sdk/master/polkadot_sdk_docs/reference_docs/chain_spec_genesis/index.html){target=_blank}, however the `build-spec` method can be utilized as well.
+
+```bash
+polkadot-parachain build-spec > myCustomSpec.json
+```
+
+After you export the chain spec, you can modify any of its fields in a text editor. For example, you might want to change the network name, bootnodes, and any genesis storage items, such as token balances. After editing the JSON file, you can start the node using the customized JSON.
+
+For example:
+
+```bash
+polkadot-parachain --chain myCustomSpec.json
+```
+
+### Using Chain Spec Builder
+
+The chain specification builder (`chain-spec-builder`) is a CLI tool that is used for generating chain specifications from the runtime of a node. Patch files may be used for genesis values, or a "baseline" chain specification can be generated:
+
+```bash
+chain-spec-builder create -v -r <path-to-wasm-runtime-here>
+```
+
+## Raw chain specifications
+
+With runtime upgrades, the blockchain's runtime can be upgraded with newer business logic. Chain specifications contain information structured in a way that can be understood by the node's runtime. For example, consider this excerpt of a common entry for a chain specification:
+
+```json
+"sudo": {
+  "key": "5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY"
+}
+```
+
+In the JSON file, this key and its associated value are human-readable text. However, this information can't be stored in this format in the underlying storage structures that Substrate uses. Before you can use the chain specification to initialize the genesis storage for a node, the human-readable keys must be transformed into actual storage keys that allow the values to be stored in the storage trie. This transformation is straight-forward, but it requires that the chain specification to be encoded in a format that node runtime can read.
+
+To enable a node with an upgraded runtime to synchronize with a chain from genesis, the human-readable chain specification is encoded in a **raw** format.
+The raw format enables you distribute chain specifications that all nodes can use to synchronize the chain even after runtime upgrades.
+
+Nodes support the `--raw` command-line option to produce the raw chain specifications.
```

Comment:
Also `chain-spec-builder` supports that by:
 `create --raw` or dedicated `convert-to-raw` command.